regolith/templates/preslist.tex

```diff
@@ -20,7 +20,7 @@
 \item {{ latex_safe(pres['authors'])-}}, {\bf {{ latex_safe(
 sentencecase(pres['title']))-}} },
 {%-set inst = pres['institution'] %}
-{%-if pres['type'] in ['colloquium', 'seminar']%}
+{%-if pres['type'] in ['colloquium', 'seminar', 'webinar']%}
```

Comment:
I guess webinar could be a seminar or a conference presentation.  I think we will have to handle it separately.  

Actually, come to think of it, it is not really a type of presentation, but a mode of delivering it, so I guess we may need a new field called ``webinar:`` that is False by default but can be set to true.    If it is true we just change some text somewhere to indicate this?  Could you play with it a bit and see if something makes sense?  Take a look at my presentation list (I will send it) and see what you think it should read for a webinar, then try to code it up.

Reply:
I tried! Can you look at this?